### PR TITLE
fix(DB/Quest): Shadow Vault Decree

### PR DESCRIPTION
**Pull request by wizzymore**
## Changes Proposed:
- Make Quest Item usable where it normally should be used.
Before:
image
After:
image
I've took in consideration exploiting this under the edge and it is not working ;D

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3284

## Tests Performed:
- Windows

## How to Test the Changes:

.go c 124058
.q a 12943
Try to use the quest item before and after

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1609118876085874200.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609118876085874200');
+
+DELETE FROM `gameobject` WHERE (`id` = 192011);
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
+(99732, 192011, 571, 0, 0, 1, 1, 8426.94, 2921.13, 606.259, 1.61609, 0, 0, -0.722938, -0.690913, 25, 0, 1, '', 0),
+(2134518, 192011, 571, 0, 0, 1, 1, 8427.42, 2910.6, 606.259, 1.61609, 0, 0, -0.722938, -0.690913, 300, 0, 1, '', 0);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1609118876085874200.sql:6]: Always delete ID & GUID

**Review by Kitzunu — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1609118876085874200.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+DELETE FROM `gameobject_template` WHERE (`entry` = 192011);
+INSERT INTO `gameobject_template` (`entry`, `type`, `displayId`, `name`, `IconName`, `castBarCaption`, `unk1`, `size`, `Data0`, `Data1`, `Data2`, `Data3`, `Data4`, `Data5`, `Data6`, `Data7`, `Data8`, `Data9`, `Data10`, `Data11`, `Data12`, `Data13`, `Data14`, `Data15`, `Data16`, `Data17`, `Data18`, `Data19`, `Data20`, `Data21`, `Data22`, `Data23`, `AIName`, `ScriptName`, `VerifiedBuild`) VALUES
+(192011, 8, 0, 'Thane Ufrang the Mighty\'s Spell Focus', '', '', '', 1, 1571, 17, 0, 1, 0, 0, -1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, '', '', -18019);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1609118876085874200.sql:10]: what are you changing in the `gameobject_template` ? why not just using an `UPDATE` in this case?

File: data/sql/updates/pending_db_world/rev_1609118876085874200.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609118876085874200');
+
+DELETE FROM `gameobject` WHERE (`id` = 192011 and `guid` IN (99732, 2134518));
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1609118876085874200.sql:3]: AND

Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1609118876085874200.sql:3]: isn't there an "AND" already?

File: data/sql/updates/pending_db_world/rev_1609118876085874200.sql (hunk 2 above)
Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1609118876085874200.sql:10]: I don't remember, i will look into this and change this to an `UPDATE`

File: data/sql/updates/pending_db_world/rev_1609118876085874200.sql (hunk 3 above)
Comment (LukasVolgger) [reply data/sql/updates/pending_db_world/rev_1609118876085874200.sql:3]: He means that your 'and' should be capitalized

Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1609118876085874200.sql:3]: Oh, thanks @LukasVolgger

File: data/sql/updates/pending_db_world/rev_1609118876085874200.sql (hunk 2 above)
Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1609118876085874200.sql]: I only changed the radius to 17, updated now.

**Review by LukasVolgger — APPROVED** [review]

**Comment by wizzymore** [comment]
Hi, i can confirm that the errors are not correct, the SpellFocus work correctly for the quest's item spell.

**Review by FrancescoBorzi — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1609118876085874200.sql [hunk 4]
```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609118876085874200');
+
+DELETE FROM `gameobject` WHERE (`id` = 192011 AND `guid` IN (99732, 2134518));
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
+(99732, 192011, 571, 0, 0, 1, 1, 8426.94, 2921.13, 606.259, 1.61609, 0, 0, -0.722938, -0.690913, 25, 0, 1, '', 0),
+(2134518, 192011, 571, 0, 0, 1, 1, 8427.42, 2910.6, 606.259, 1.61609, 0, 0, -0.722938, -0.690913, 300, 0, 1, '', 0);
+
+UPDATE `gameobject_template` SET `Data1` = 17 WHERE (`entry` = 192011);
+
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1609118876085874200.sql:9]: Remove one line at the end =)

**Review by Kitzunu — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1609118876085874200.sql (hunk 4 above)
Comment (wizzymore) [reply data/sql/updates/pending_db_world/rev_1609118876085874200.sql]: Removed now, on the default ubuntu editor there was no other line, so i deleted it with VSC, it sucks that everywhere is a different format

**Comment by Voxstrasza** [comment]
looks good to me, thanks for this PR

**Comment by LukasVolgger** [comment]
Thx @Voxstrasza for testing und @wizzymore for this good job!

**Comment by pklloveyou** [comment]
Tested

**Comment by Kitzunu** [comment]
Thanks =)

**Pull request merged**